web/src/lib/components/elements/buttons/button.svelte

```diff
@@ -26,6 +26,7 @@
   export let fullwidth = false;
   export let border = false;
   export let title: string | undefined = '';
+  export let ref: HTMLButtonElement | null = null;
```

Comment:
What is this used for?

Reply:
I used it to focus on the button from the parent component.